src/js/control/select.js

```diff
@@ -195,6 +195,67 @@ export default class controlSelect extends control {
       otherInputValue.style.display = 'none';
     }
   }
+
+
+    /**
+   * onRender callback
+   */
+  onRender() {
+    // Set userData if available
+    if(this.config.userData){ 
+      let selectedOptions = [];
+      for (let i = 0; i < this.config.userData.length; i++) {
+        selectedOptions.push(this.config.userData[i]);
```

Comment:
could this loop be replace with `selectedOptions = this.config.userData.slice()`?

Reply:
Yes I did slice(0) and it seems to work fine.